apk_build/APKBUILD

```diff
@@ -0,0 +1,81 @@
+pkgname=stronglahav
+pkgver=1.0.0
+pkgrel=1
+pkgdesc="StrongLahav"
+url="https://www.strongswan.org/"
+arch="aarch64"
+pkgusers="ipsec"
+pkggroups="ipsec"
+license="GPL-2.0-or-later WITH OpenSSL-Exception"
+options="!check" # failing tests
```

Comment:
Why the tests are failing?

Reply:
Feel free to check and fix, I won't mind:

```Passed all 2 'prf_plus' test cases
  Running suite 'http fetcher':
    Running case 'GET': +++-
      Failure in 'test_get': service != NULL (suites/test_fetch_http.c:211, i = 3)
    Running case 'POST': ++-
      Failure in 'test_post': service != NULL (suites/test_fetch_http.c:250, i = 2)
    Running case 'response code': +++++
  Passed 1/3 'http fetcher' test cases
  Running suite 'ed25519':
    Running case 'ed25519_sign': +++++
    Running case 'ed25519_gen': +
    Running case 'ed25519_fail': +
    Running case 'ed25519_speed': +
  Passed all 4 'ed25519' test cases
  Running suite 'ed448':
    Running case 'ed448_sign': ++++++++
    Running case 'ed448_gen': +
    Running case 'ed448_fail': +
    Running case 'ed448_speed': +
  Passed all 4 'ed448' test cases
  Running suite 'signature params':
    Running case 'rsa/pss parse': +++++++++++++++
    Running case 'rsa/pss build': +++++++++++++
    Running case 'rsa/pss salt len': ++++++++++
    Running case 'params compare': +++++++++
    Running case 'params clone': ++++++
    Running case 'params clear': ++++++
    Running case 'parse': +++++++++++++++++++
    Running case 'build': ++++++++++++++++
  Passed all 8 'signature params' test cases
  Running suite 'metadata':
    Running case 'integer types': ++++++
  Passed all 1 'metadata' test cases
  Running suite 'metadata_set':
    Running case 'create/destroy': ++
    Running case 'put/get': +++++++++
    Running case 'clone': +++
  Passed all 3 'metadata_set' test cases
Passed 46 of 49 'libstrongswan' suites
FAIL: libstrongswan_tests
==================
1 of 1 test failed
==================
```

Reply:
This is your PR.....

Reply:
... where I do not have functional code changes. I'm just doing the build.

Reply:
same test failures in `main` branch with `86_64` build, so I rest my case on this